Watchman.Discord/Areas/Administration/Controllers/AdministrationController.cs

```diff
@@ -116,5 +118,120 @@ public async Task SetRoleAsSafe(DiscordRequest request, Contexts contexts)
             var messagesService = _messagesServiceFactory.Create(contexts);
             await messagesService.SendResponse(x => x.RoleSettingsChanged(roleName), contexts);
         }
+
+        [AdminCommand]
+        [DiscordCommand("add response")]
+        public async Task AddResponse(DiscordRequest request, Contexts contexts)
```

Comment:
Zrób nowe Areas "Responses"
wrzuć tam ResponsesController

myśle że przydałby sie serwis z `GetResponseByOnEvent(string onEvent, ulong serverId = 0)`

Reply:
W sporej części by to wyczyściło kod, bo ta akcja powtarza sie kilka razy

Nazwij go np ResponsesService